app/assets/stylesheets/navbar.css

```diff
@@ -1,39 +1,30 @@
 #nav-bar {
     position: fixed;
-    padding: var(--navbar-padding);
     top: 0;
     left: 0;
     right: 0;
     height: var(--navbar-height);
 
     display: grid;
     grid-template-columns: 1fr 1fr 1fr;
-
-    background-image: linear-gradient(white, rgba(255, 255, 255, 0.3));
+    
     border-bottom: 1px solid rgba(255, 255, 255, 0.9);
-    backdrop-filter: blur(calc(120 * var(--fpx)));
-    -webkit-backdrop-filter: blur(calc(120 * var(--fpx)));
+    backdrop-filter: blur(calc(270 * var(--fpx)));
+    -webkit-backdrop-filter: blur(calc(270 * var(--fpx)));
+
+    user-select: none;
 
-    z-index: 2;
+    z-index: 200;
```

Comment:
Sorry for the confusion, just thought about making it larger in case there will be some other elements inserted between navbar and contents behind. Maybe it doesn't make sense to you so I'll change it back.

Reply:
Ah I see, that makes sense :) it's not a problem if you wanted to leave it